packages/vue/src/components/shared/DropDown.jsx

```diff
@@ -168,6 +168,7 @@ const Dropdown = {
 															count: item.doc_count,
 															isChecked:
 																selected && this.$props.multi,
+															data: item,
```

Comment:
This won't work currently as the object will be `label` & `count` and we want all the data.